google-cloud-storage/src/test/java/com/google/cloud/storage/DefaultRetryHandlingBehaviorTest.java

```diff
@@ -149,7 +152,7 @@ public static Collection<Object[]> testCases() {
 
     // ensure all permutations are accounted for, reporting any that haven't been and providing
     // a stub which can be used to easily define them.
-    assertWithMessage("Missing mappings for tokens").that(difference).isEmpty();
+    // assertWithMessage("Missing mappings for tokens").that(difference).isEmpty();
```

Comment:
are you planning on removing this line?

Reply:
This was an accident, I'll revert this line.